### DISCUSSION TITLE
Org template - filter/remove projects

### DISCUSSION THREAD

**Discussion by neensKB** [nbrown02/FlowViz#116] [General]
Hi! We have over 100 projects in our ADO org, some of which aren't in use anymore and about 20 are used by other departments (working on addressing both of these things...) - is there a way to filter out projects prior to attempting to load the data? I'm getting notifications inside ADO that I'm exceeding rate limiting thresholds 😅

I'm still finding my way around PBI and learning to read/amend queries so would reatly appreciate a pointer in the right direction :)

**Comment by nbrown02** [comment]
You can modify the queries to select particular projects - check #107 for the discussion on this :)
